app/services/edge_cache/bust_comment.rb

```diff
@@ -0,0 +1,32 @@
+module EdgeCache
+  class BustComment < Bust
+    def self.call(commentable)
+      return unless commentable
+
+      bust_article_comment(commentable) if commentable.is_a?(Article)
+      commentable.touch(:last_comment_at) if commentable.respond_to?(:last_comment_at)
+
+      bust("#{commentable.path}/comments/")
+      bust(commentable.path.to_s)
+
+      commentable.comments.includes(:user).find_each do |comment|
+        bust(comment.path)
+        bust("#{comment.path}?i=i")
+      end
+
+      bust("#{commentable.path}/comments/*")
+    end
+
+    # bust commentable if it's an article
+    def self.bust_article_comment(commentable)
```

Comment:
Do you want this to be part of the public interface or should we add a `private_class_method :bust_article_comment`?